### ISSUE TITLE
SPIKE: Send sample HIV message to ReportStream

### ISSUE THREAD

**Issue by lg-king** [CDCgov/prime-simplereport#6069]
## Why should we do this?
We need to ensure that LAC can receive and use HIV test results for our pilot to be successful.

## Action requested
Work with RS to send a sample HIV message (FHIR format) to CA and LAC. We should send both positive and negative, as there may be different requirements for each. 

You can do this from our `test` environment. They may be able to do this for themselves actually, but someone needs to work with them in case they have any questions or need help setting up an HIV device. We might also need to turn on the single-entry HIV feature flag for the environment.

## Acceptance criteria
_are these still valid AC?_
- CA can receive positive and negative HIV messages and route to LAC 
- LAC can receive and view positive and negative sample HIV test result with all required fields 

## Additional context
Related to #5538
Slack thread with docs: https://skylight-hq.slack.com/archives/C02HKNHGCP6/p1688656583617709
- Do this after HIV fhir converter is fully implemented

**Comment by mehansen** [comment]
blocked by #6582

**Comment by mehansen** [comment]
Sent a sample HIV message from `test` and confirmed with Victor from RS that the message got processed and converted fine and that the new AOEs were appearing properly in the message.

Victor pointed out that we were missing the LOINC long names, which is because the devices in the `test` db were never populated with this data. Created #7460 to add those and resend a test message to confirm that part.

Further testing for routing from ReportStream to CA and LAC need to be done by ReportStream, so are out of scope of this ticket. Closing as complete.
